homeassistant/components/zha/binary_sensor.py

```diff
@@ -291,6 +291,17 @@ class XiaomiPlugConsumerConnected(BinarySensor, id_suffix="consumer_connected"):
     _attr_device_class: BinarySensorDeviceClass = BinarySensorDeviceClass.PLUG
 
 
+@MULTI_MATCH(
+    cluster_handler_names="tuya_manufacturer",
+    quirk_classes={"zhaquirks.tuya.ts0601_trv.ZonnsmartTV01_ZG"},
```

Comment:
Due to https://github.com/home-assistant/core/pull/93268, this would need to be updated to `ts0601_trv.ZonnsmartTV01_ZG` everywhere, right?

Reply:
No, the change in #93268 will compare only the 2 last tokens from both the component's quirk_class and the match definition. Everything else is not taken into account.